types/inet.go

```diff
@@ -135,24 +135,30 @@ func (a InetArray) String() string {
 	return o.String()
 }
 
-func (a *InetArray) ValueStr(i int) string {
+func (a *InetArray) Value(i int) *net.IPNet {
+	if a.IsNull(i) { // IsValid ~ !IsNull
```

Comment:
I'm not 100% sure what this comment is trying to say; is it saying `IsValid is approximately the same as !IsNull` ? Why not just use `IsNull` and leave it at that?

Reply:
I saw several times that we used `if a.IsNull(i) || !a.IsValid(i)` and wanted to express that those are, in fact, just opposites (judging by the code, the following is true: `a.IsNull(i) == !a.IsValid(i)` for every `i`

Reply:
Okay, yeah it's fine to remove it I think. We should just go and clean up the other spots in the cloudquery codebase so the `!IsValid()` version doesn't keep getting copy-pasted around